packaging/Slackbuild/rmw/README.Slackware

```diff
@@ -1,4 +1,4 @@
-# rmw-0.7.04
```

Comment:
This file is just a copy of the main README.md file and gets included when I prepare a Slackbuild package